src/sonic-yang-models/yang-models/sonic-bgp-bbr.yang

```diff
@@ -0,0 +1,37 @@
+module sonic-bgp-bbr {
+    namespace "http://github.com/sonic-net/sonic-bgp-bbr";
+    prefix bgpbbr;
+    yang-version 1.1;
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC BGP BBR";
+
+    revision 2023-12-25 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-bgp-bbr {
+        container BGP_BBR {
+
+            description "BGP_BBR table part of config_db.json";
+
+            container all {
+                leaf status {
+                    type string {
+                        pattern "enabled|disabled";
```

Comment:
We can use admin_mode for this :
https://github.com/sonic-net/sonic-buildimage/blob/d436dd6d84db05b7a1833bb5a1482a8763afcd3b/src/sonic-yang-models/yang-templates/sonic-types.yang.j2#L127

Reply:
Others lgtm